modules/ROOT/pages/design-add-api-dependency.adoc

```diff
@@ -62,10 +62,31 @@ When you have your branch, a pop-up appears where you can see the missing rulese
 _Result_: The rulesets are added to your project.
 
 
-[fix-gov-issues]
+[[fix-gov-issues]]
 == Fix Governance Conformance Issues
 
-After you add the rulesets as dependencies, you can view the conformance messages in the *Project Errors* section of the text editor. Use the provided information to fix any conformance issues and then republish your API specification. 
+When adding the rulesets as dependencies, you can view the conformance messages in the *Project Errors* section of the text editor. Use the provided information to fix any conformance issues and then republish your API specification. 
```

Comment:
Shouldn't we say `governance validations` instead of `conformance messages`?

Reply:
@ignaciovazgueza, we should be consistent with the API Governance terminology, as this is part of the API Governance user journey. APIs are either conformant or nonconformant to rulesets.

Reply:
Also, @failup, I noticed you changed this to "When adding" instead of "After you add" the rulesets as dependencies. The dependencies have to actually be added to see the errors appear in the Project Errors page, right, or has the timing on that changed? My understanding is that the rulesets are added when they click the plus icon and that dependency is auto-saved in APID. Are you interpreting this otherwise? If so, we should probably explain more here.

Reply:
Looking closer at this, I think the issue is that the validation is put in the wrong section. This either needs to be in its own section above Fix Governance Conformance Issues, or it needs to be integrated more fully into the section and the name of the section changed to encompass both running the validation and fixing the conformance.

Reply:
Also, as we discussed in our sync last week, I suggest reorganizing this new content so that it has a simpler lead-in and has the "what to do" in a procedure rather than spread out into paragraphs.
The validation is done automatically by default. If you get a message that the validation is taking too long:
1) Make the following changes to improve performance:
 (list things to do to avoid the situation here instead of in a troubleshooting section)
2) Click *Validate for Governance*.
...

Reply:
Is there any point of them retrying (clicking Ok when they get the error) before taking steps to improve the performance? If so, you could put that as the first step in the procedure followed by the ones I mention above (the steps are paraphrased to show organization suggestions, not specific text suggestions).